_pages/contributing/git-flow-and-releases.md

```diff
@@ -0,0 +1,122 @@
+
+## Git flow
+
+Flow in spartacus project was structured around version support described above.
+
+`develop` is the default branch intended for new version development (is used for minor/major versions). All features and bug fixes are merged to this branch.
+
+There is also so called maintenance branch which changes on new `stable`/`rc` release and is used for patch versions. Only bug fixes are merged to this branch.
+Once we release version `1.4.0-rc.0` the `release/1.4.x` branch is treated as a maintenance branch. When we release `1.5.0-rc.0` version then the `release/1.5.x` branch becomes the maintenance branch and so on.
+
+Other branches convention:
+
+- `feature/GH-xxxx` branches are used for simple features and bug fixes
+- `epic/epic-name` branches are used for big features (called epics)
+- `release/1.4.0-rc.0` branches are used for specific release (you can distinguish them from maintenance branches, because the full version number is included)
+
+### Flow for epic development
+
+- create from `develop` branch new `epic/epic-name` branch
+- create branches for epic subtasks from `epic/epic-name` and merge them to `epic/epic-name` branch
+- from time to time update your `epic/epic-name` branch with changes from `develop` branch (it will help you with managing conflicts)
+- once the epic is complete create PR and merge `epic/epic-name` branch to `develop` branch
+
+### Flow for new simple feature
+
+- create from `develop` branch new `feature/GH-xxxx` branch
+- develop your feature
+- when you finish create PR and merge `feature/GH-xxxx` branch to `develop` branch
+
+### Flow for bug fix
+
+- create from `develop` branch new `feature/GH-xxxx` branch
+- fix the bug
+- create PR and merge `feature/GH-xxxx` branch to `develop` branch
+- if this fix applies to actively supported version create new branch `feature/GH-xxxx-maintenance` from maintenance branch
+- cherry pick commit with the fix from `develop` branch
+- create PR and merge `feature/GH-xxxx-maintenance` into maintenance branch
+
+## Release schedule
+
+Currently we don't have scheduled releases. Product owner or team decides when is a good time to release a new version.
+
+Terms we currently use that are misleading:
+
+- feature freeze - describes moment when we completed all features for new minor/major release (means that we want to release `rc` very soon, but still needs to fix some bugs)
+- code freeze - describes moment when you should stop committing code (that is not needed with our flow, because we can always cut release/maintenance branch and keep committing)
+
+Replacements for those terms:
+
+- feature freeze -> new maintenance branch and release new `rc` - first RC can be buggy, that's the point of RC
+- code freeze -> create new release branch - don't block main development/maintenance branch ever (don't need to bother developers with these details, because our flow supports concurrent work on these branches and releasing another version).
+
+## Release steps
```

Comment:
I think the whole section on release steps can be dropped.

Reply:
Done 🙂